src/main/java/io/strimzi/kafka/bridge/tracing/TracingUtil.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+package io.strimzi.kafka.bridge.tracing;
+
+import io.strimzi.kafka.bridge.config.BridgeConfig;
+import io.vertx.ext.web.RoutingContext;
+import io.vertx.kafka.client.producer.KafkaProducerRecord;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Map;
+import java.util.Properties;
+
+/**
+ * Tracing util to hold app's Tracing instance.
+ */
+public class TracingUtil {
+    private static final Logger log = LoggerFactory.getLogger(TracingUtil.class);
+    private static TracingHandle tracing = new NoopTracing();
+
+    public static TracingHandle getTracing() {
+        return tracing;
+    }
+
+    public static void initialize(BridgeConfig config) {
+        String tracingConfig = config.getTracing();
+        if (tracingConfig != null && tracingConfig.contains("jaeger")) {
+            boolean isOpenTelemetry = "jaeger-otel".equals(tracingConfig);
+            TracingHandle instance = isOpenTelemetry ? new OpenTelemetryHandle() : new OpenTracingHandle();
+
+            String serviceName = instance.serviceName(config);
+            if (serviceName != null) {
+                log.info(
+                    "Initializing Jaeger ({}) tracingConfig with service name {}",
+                    isOpenTelemetry ? "OpenTelemetry" : "OpenTracing",
+                    serviceName
+                );
```

Comment:
Does this make sense? Doesn't `serviceName` ecist only for OTel? I.e. would this log line ever print for OpenTelemetry?

Reply:
No, it exists for both -- this is the unique name for your traces / spans (e.g. when you check them in the Jaeger server UI).